cortx-monitorQuickstartGuide.md

```diff
@@ -103,57 +87,116 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
 
 2. Build RPMs
     
-   1. Switch to the directory where you've cloned cortx-monitor and run the command:
+   1. Checkout the main branch
    
-        `$ jenkins/build.sh`  
+        `docker run --rm -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make checkout BRANCH=main`  
     
     :page_with_curl: **Notes:** 
     
-    - It takes 2-3 minutes to generate RPMs.
-    - Once the RPMs are successfully generated, you can view it from:
-    
-        `/root/rpmbuild/RPMS/noarch/` and,
-     
-        `/root/rpmbuild/RPMS/x86_64/`
-
-    **Sample Output:**
+    - If you get below error it may be because that your git version may be more thatn 2.x. Cortx requires 1.8.3.1. In that case
+      if go to `/root/cortx/cortx-monitor/` and run `git checkout main` 
```

Comment:
i can still still /root references ..
can we change to 
~/cortx/cortx-monitor, so it would apply for any user, including root. test though

Reply:
Done.